unlock-app/src/components/content/certification/CertificationContent.tsx

```diff
@@ -23,7 +23,7 @@ export const CertificationContent = () => {
   return (
     <AppLayout showLinks={false} authRequired={false} title="">
       <Head>
-        <title>{pageTitle('Event')}</title>
+        <title>{pageTitle('Certification')}</title>
```

Comment:
Can we add here the actual name of the certification if we're loading one?

Reply:
Added!